php/src/app/handler/Post/CreatePostHandler.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Handler\Post;
+
+use App\Repo\DBClient;
+use App\Repo\PostRepo;
+
+class CreatePostHandler
+{
+    private \PDO $pdo;
+    function __construct()
+    {
+        //pdoの初期化
+        try{
+            $this->pdo = DBClient::init_pdo();
+        }catch (\Exception $e){
+            throw $e;
+        }
```

Comment:
ここは結局throwしているので、わざわざtry~catchする必要は無いですね。

例えばPHPDocでHintを書くというやり方があります
```suggestion
    /**
     * @throws \Exception
     */
     function __construct()
    {
        //pdoの初期化
        $this->pdo = DBClient::init_pdo();
```